tools/stress-cluster/services/otelcollector/otel-collector-config.yml

```
@@ -0,0 +1,33 @@
+receivers:
+  otlp:
+    protocols:
+      http:
+      grpc:
+ 
+processors:
+  batch:
+ 
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [logging, azuremonitor]
+    metrics:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [prometheus, azuremonitor]
+    logs:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [azuremonitor]
+
+exporters:
+  prometheus:
+    endpoint: "0.0.0.0:8889"
+    resource_to_telemetry_conversion: 
+      enabled: true
+  logging:
+    verbosity: detailed
```

```suggestion
    verbosity: normal
```

This reduces the noise level in the collector dramatically.

---

Done!